docs/customization/customizing-your-storefront.mdx

```diff
@@ -0,0 +1,170 @@
+---
+sidebar_position: 1
+---
+
+# Customizing your storefront
+
+In this guide, we're going to show you an example of how to customize your storefront. In
+particular, we're going to add a hero image to the hero section of the home page:
+
+![Hero image placeholder](/img/customizing-your-storefront/hero-image-placeholder.png)
+
+## Installing the frontend
+
+With Solidus 3.2, installing Solidus would automatically install[SolidusStarterFrontend][], which is
```

Comment:
Missing link for SolidusStarterFrontend? Or this is just a syntax I don't know?

Reply:
It's an... "implicit link name shortcut" :p From https://daringfireball.net/projects/markdown/syntax#link:

![image](https://user-images.githubusercontent.com/61476/184312614-f00275db-6ed2-4515-b623-d4ac1025bc41.png)

I should fix that space between "install"  and the link, though :)

Reply:
@gsmendoza FYI this doesn't seem to be working in the [preview](https://2c3489d1.edgeguides.pages.dev/customization/customizing-your-storefront), so it probably needs to be explicit.

Reply:
@aldesantis I forgot to add the link for SolidusFrontend :p Fixing it...